test/common/http/http1/codec_impl_test.cc

```diff
@@ -908,6 +908,67 @@ TEST_P(Http1ServerConnectionImplTest, Http10MultipleResponses) {
   }
 }
 
+// SPELLCHECKER(off)
+struct {
```

Comment:
structs in Envoy members have _ suffix: https://github.com/envoyproxy/envoy/blob/main/STYLE.md#deviations-from-google-c-style-guidelines

nit: consider defining struct first then adding the list of cases. We can also push the list of cases, the struct into the test itself since no other tests use it to better encap it.

Here and below